safe/gui/tools/shake_grid/test/test_shakemap_converter_dialog.py

```diff
@@ -44,13 +44,13 @@ def test_init_dialog(self):
         msg = 'Dialog is failed to create'
         self.assertIsNotNone(shakemap_converter_dialog, msg)
 
+    @unittest.skip("TESTDATA doesn't exist anymore.")
     def test_behaviour(self):
-        """Test behaviour of elements in the dialog
-        """
+        """Test behaviour of elements in the dialog."""
         shakemap_importer_dialog = ShakemapConverterDialog(PARENT, IFACE)
         shakemap_importer_dialog.use_output_default.setEnabled(True)
-        my_grid_path = os.path.join(TESTDATA, 'grid.xml')
```

Comment:
I guess this not was not running because the name file doesn't start by 'test'. I can't find the grid.xml in the inasafe/test_data repo.